assets/sass/components/banner/_googlesitekit-banner.scss

```
@@ -152,3 +158,7 @@
 		}
 	}
 }
+
+.googlesitekit-banner__setup-cta {
```

I think this should be a "modifier" instead of an element that depends on the `googlesitekit-banner` "block". 
```suggestion
.googlesitekit-banner--setup-cta {
```